app/controllers/completed_tasks_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CompletedTasksController < ApplicationController
+  before_action :authenticate_user!
+  def new
+  	@ct = CompletedTask.new
+  end
+  def create
+  	if(params[:is_checked].to_s == "true")
+  		current_user.completed_tasks.build(todo_item_id: params[:completed_task]["todo_item_id"])
+  	else
+  		current_user.completed_tasks.where(todo_item_id: params[:completed_task]["todo_item_id"]).destroy_all
+  	end
+      if current_user.save
+      	respond_to do |format| 
+        format.html { redirect_to root_url, notice: "Save process completed!" }
+        format.json { render json: nil, status: :ok }
+    	end
+      else
+      	respond_to do |format| 
+          format.json { 
```

Comment:
Avoid using {...} for multi-line blocks.<br>Use 2 (not 5) spaces for indentation.<br>Trailing whitespace detected.